### PR TITLE
fix typos in welcome project

### PR DESCRIPTION
**Pull request by Ben9986**
There are two small typos in the "Meet Planify" project. This commit fixes them :)

### CODE REVIEW

**Comment by alainm23** [comment]
Hello and thanks for the correction, the correct flow would be to first edit the source code, then generate the new translation files and fix the translation file if necessary.

1. Edit [this](https://github.com/alainm23/planify/blob/master/src/Util.vala#L1167) code line
2. Generate the new translation files using the command `ninja io.github.alainm23.planify-update-po`inside the build folder
3. Check if the translation files were updated correctly.

I understand that it is a long process, but just editing the source code file is enough. thanks.

File: po/io.github.alainm23.planify.pot [hunk 1]
```diff
@@ -334,7 +334,7 @@ msgstr ""
 
 #: src/Util.vala:1165
 msgid ""
-"To reorder your list, tap amd hold a to-do, then drag it to where it shpuld "
+"To reorder your list, tap and hold a to-do, then drag it to where it should "
```
Comment (alainm23) [inline po/io.github.alainm23.planify.pot]: Edit source code instead of translations

**Review by alainm23 — CHANGES_REQUESTED** [review]

File: po/io.github.alainm23.planify.pot (hunk 1 above)
Comment (Ben9986) [reply po/io.github.alainm23.planify.pot]: sorry! unfamiliar with the project and admittedly didn't look hard enough for the correct file. I've edited the source file now

**Comment by Ben9986** [comment]
I can't seem to get the build working on my setup. Can you check it and lmk if I've missed something

**Comment by alainm23** [comment]
Don't worry, this is enough, thank you.

**Pull request merged**